src/libasr/codegen/asr_to_c_cpp.h

```diff
@@ -1664,12 +1664,9 @@ R"(#include <stdio.h>
         if (!c) {
             increment = 1;
         } else {
-            if (c->type == ASR::exprType::IntegerConstant) {
-                increment = ASR::down_cast<ASR::IntegerConstant_t>(c)->m_n;
-            } else if (c->type == ASR::exprType::IntegerUnaryMinus) {
-                ASR::IntegerUnaryMinus_t *ium = ASR::down_cast<ASR::IntegerUnaryMinus_t>(c);
-                increment = - ASR::down_cast<ASR::IntegerConstant_t>(ium->m_arg)->m_n;
-            } else {
```

Comment:
This should be extended to support runtime increment values. Opened https://github.com/lcompilers/lpython/issues/1392 for discussion.